src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -45,17 +45,44 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
   if (!highlighted) {
     str = markdownIt.utils.escapeHtml(str);
   }
+  let highlightLines = token.attrGet('highlight-lines')
```

Comment:
You have missing `;`s in this file.

Reply:
Done. Turns out eslint ignores `src/lib/markbind/src/lib/markdown-it/*` 😱